bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/widgets/Display.java

```diff
@@ -3130,6 +3122,26 @@ Rectangle mapInPixels (Control from, Control to, int x, int y, int width, int he
 	return new Rectangle (rect.left, rect.top, rect.right - rect.left, rect.bottom - rect.top);
 }
 
+Point translateFromDisplayCoordinates(Point point, int zoom) {
+	return coordinateSystemMapper.translateFromDisplayCoordinates(point, zoom);
+}
+
+Point translateToDisplayCoordinates(Point point, int zoom) {
+	return coordinateSystemMapper.translateToDisplayCoordinates(point, zoom);
+}
+
+Rectangle translateFromDisplayCoordinates(Rectangle rect, int zoom) {
+	return coordinateSystemMapper.translateFromDisplayCoordinates(rect, zoom);
+}
+
+Rectangle translateToDisplayCoordinates(Rectangle rect, int zoom) {
+	return coordinateSystemMapper.translateToDisplayCoordinates(rect, zoom);
+}
+
+private int getApplicableMonitorZoom(Monitor monitor) {
+	return DPIUtil.getZoomForAutoscaleProperty(isRescalingAtRuntime() ? monitor.zoom : getDeviceZoom());
+}
```

Comment:
This method only seems to be used within the `MultiZoomCoordinateSystemMapper`, so maybe we could move it there and remove the case distinction whether rescaling at runtime is active (as that mapper is only used when that mode is active).

Reply:
Good point, I moved and adapted the method accordingly